Two additional functions for handling VM snapshots

MatsBEkman opened the pull request:
<!-- Please submit Pull Requests to the Devel branch. No Pull Requests are accepted to the Master branch -->

# Description

Added 2 new functions for handling downloadlinks and download from VM Snapshots

## Related Issue

This project only accepts pull requests related to open issues.

* If suggesting a new feature or change, please discuss it in an issue first.
* If fixing a bug, there should be an issue describing it with steps to reproduce

[[_Please link to the issue here_](https://github.com/rubrikinc/rubrik-sdk-for-powershell/issues/820)]

## Motivation and Context

Automatic restores from VM snapshots

## How Has This Been Tested?

Tested together with HCL Domino integration for backups and restore of Domino databases

## Screenshots (if appropriate):

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply. If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [X ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **[CONTRIBUTION](https://rubrik.gitbook.io/rubrik-sdk-for-powershell/user-documentation/contribution)** document.
- [ ] I have updated the CHANGELOG file accordingly for the version that this merge modifies.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

mwpreston reviewed (approved):
Looks good to me - I've added a couple of Pester tests and updated changelog - tested locally  here and everything works!  Thanks for the contribution!

MatsBEkman commented:
Great thanx
I will download now and test in production at the bank
Great news
Time to proceed

Regards
Mats

fre 24 feb. 2023 kl. 19:44 skrev Mike Preston ***@***.***>:

> ***@***.**** approved this pull request.
>
> Looks good to me - I've added a couple of Pester tests and updated
> changelog - tested locally here and everything works! Thanks for the
> contribution!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rubrikinc/rubrik-sdk-for-powershell/pull/828#pullrequestreview-1314089251>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALQXB63SYHNDI6THGB62ICDWZD6SNANCNFSM6AAAAAAUO47VVI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Med Vänlig Hälsning
Mats
Regards
Mats
